airbyte-integrations/connectors/source-github/source_github/streams.py

```diff
@@ -721,6 +721,9 @@ def parse_response(self, response: requests.Response, **kwargs) -> Iterable[Mapp
                 record["repository"] = self._get_name(repository)
                 if record["merged_by"]:
                     record["merged_by"]["type"] = record["merged_by"].pop("__typename")
+                # to avoid conflict with old field 'mergeable' of type 'boolean' in REST API
+                if "mergeable" in record:
```

Comment:
@grubberr how did this conflict happen? did Github make a backwards breaking change in their API suddenly?

Reply:
This happened because [switching](https://github.com/airbytehq/airbyte/pull/13763) from REST API -> GraphQL API for this stream `pull_request_stats`.  It looks like Github provide more extended values for this field `mergeable` in GraphQL API.

I provide you schema change for convenience:

OLD REST API SCHEMA

| Type      | Name                         |  possible values |
| -------- | -------------------- |----------
| boolean | mergeable                 | false, null
| boolean | rebaseable                | false, null
| string     | mergeable_state | "dirty", "unknown"

NEW GRAPHQL SCHEMA:

| Type      | Name                          |  possible values |
| -------- | -------------------- |----------------------------------------------------
| string     | mergeable                 | "CONFLICTING", "MERGEABLE", "UNKNOWN"
| boolean | can_be_rebased       | true, false
| string     |  merge_state_status | "BLOCKED", "DIRTY", "UNKNOWN", "UNSTABLE"

Reply:
@bazarnov changed